README.md

```diff
@@ -3,6 +3,7 @@
 [![Build Status](https://travis-ci.org/prasadtalasila/IRCLogParser.svg?branch=dev)](https://travis-ci.org/prasadtalasila/IRCLogParser) 
 [![Maintainability](https://api.codeclimate.com/v1/badges/211e8675682e2d345d8b/maintainability)](https://codeclimate.com/github/prasadtalasila/IRCLogParser/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/211e8675682e2d345d8b/test_coverage)](https://codeclimate.com/github/prasadtalasila/IRCLogParser/test_coverage) 
+[![Integration test coverage](https://coveralls.io/repos/github/kaushik-rohit/IRCLogParser/badge.svg?branch=master)](https://coveralls.io/github/kaushik-rohit/IRCLogParser?branch=dev)
```

Comment:
`[![Integration test coverage](https://coveralls.io/repos/github/prasadtalasila/IRCLogParser/badge.svg?branch=dev)](https://coveralls.io/github/prasadtalasila/IRCLogParser?branch=dev)`

Reply:
@kaushik-rohit We are reporting functional tests here.

Reply:
Given that we are going to start with integration tests after unit tests, it is better to setup a reporting for integration tests as well. The integration tests go to codeclimate. Hence we can use the existing badge that purpose.